Make things comparable

currently GE ~ SNP uses BY, whilst GE ~ meth(20k) and GE ~ SNP + meth (full) both use BH

So: run pvals on genome and remake pvalue plots, again for GE ~ SNP using BH

remake plots per chr arm

run 3way model using window for methylation

check scripts for miR-mRNA model, run

dichotomize results according to FDR, plot overlapping regions

test effect of confounder: compare fit with and without confounder - in this case we can use stage